cpu/esp32/esp-wifi/esp_wifi_netdev.c

```diff
@@ -74,12 +74,22 @@ esp_err_t _esp_wifi_rx_cb(void *buffer, uint16_t len, void *eb)
 {
     DEBUG("%s: buf=%p len=%d eb=%p\n", __func__, buffer, len, eb);
 
-    CHECK_PARAM_RET (buffer != NULL, -EINVAL);
-    CHECK_PARAM_RET (len <= ETHERNET_DATA_LEN, -EINVAL);
+    if (buffer == NULL || len >= ETHERNET_DATA_LEN) {
+        if (eb) {
```

Comment:
mhm a bit of inconsistent code usage/style, i.e. above you've `buffer == NULL` which could be `!buffer` or this one should be `eb != NULL` 😄